spec/views/events/form_examples.rb

```diff
@@ -34,6 +34,24 @@
     end
   end
 
+  if with.include? :player_type
+    it 'has input for player_type' do
+      render
+
+      expect(rendered).to have_field(Event.human_attribute_name :player_type)
+    end
+
+    it 'has default input for player_type field' do
+      render
+
+      if path == :edit
+        expect(rendered).to have_field(Event.human_attribute_name :player_type, with: t(Event.human_player_type(@event.player_type)))
+      elsif path == :new
+        expect(rendered).to have_field(Event.human_attribute_name :player_type, with: t('events.new.select_player_type'))
```

Comment:
not a huge fan of splitting it up like this, the `with:` is omitted for all other tests. But I do see the value of specifically testing the default input, since it is exactly what this issue is about. We should just consider splitting the :new and :edit tests into two different sharde_examples at some point, since otherwise this one will look like one huge switch case.
Fine for now though.